app/controllers/preprints/provider/preprint-detail.js

```diff
@@ -126,7 +126,7 @@ export default Controller.extend({
         },
         submitDecision(trigger, comment, filter) {
             this.toggleProperty('savingAction');
-
+            this.set('userHasEnteredReview', false);
```

Comment:
Would it be better to change this property after the review has been successfully saved?

Reply:
`savingAction` is the property keeping track of that, right? I do not see much benefits from that change. (in case of promise failure) carrying over state changes after the corresponding action have completed might cause issues if the property is issued in some other ways.

Reply:
My only concern would be if the user tries to save their review and failed.  As I'm understanding it, `userHasEnteredReview` would still be changed to `false` even though the user's review wasn't saved, and the state of dirty fields for transitions could get a bit messed up because it's using that property as it's base.

Reply:
@baylee-d, I see what you mean. Thanks.